src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1648,14 +1648,18 @@ export class StudyViewPageStore {
                     // TODO this.barChartFilters.length > 0 ? 'STATIC' : 'DYNAMIC' (not trivial when multiple filters involved)
                     const dataBinMethod = DataBinMethodConstants.STATIC;
                     let result = [];
-                    if (this.isInitialFilterState && isDefaultAttr && !this._clinicalDataIntervalFilterSet.has(uniqueKey)) {
+                    const attribute = this._clinicalDataBinFilterSet.get(getClinicalAttributeUniqueKey(chartMeta.clinicalAttribute!))!
+                    if (this.isInitialFilterState &&
+                        isDefaultAttr &&
+                        !this._clinicalDataIntervalFilterSet.has(uniqueKey) &&
+                        attribute.disableLogScale === false) {// check if log scale is changed from default value
```

Comment:
Could you check disableLogScale !== true? So you don't have to specify disableLogScale in the initial loading